bundles/org.openhab.binding.ring/src/main/java/org/openhab/binding/ring/handler/AccountHandler.java

```diff
@@ -0,0 +1,634 @@
+/*
+ * Copyright (c) 2010-2025 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.ring.handler;
+
+import static org.openhab.binding.ring.RingBindingConstants.*;
+
+import java.io.File;
+import java.io.IOException;
+import java.math.BigDecimal;
+import java.net.InetAddress;
+import java.net.NetworkInterface;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.ScheduledFuture;
+import java.util.concurrent.TimeUnit;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+import org.openhab.binding.ring.internal.RestClient;
+import org.openhab.binding.ring.internal.RingAccount;
+import org.openhab.binding.ring.internal.RingDeviceRegistry;
+import org.openhab.binding.ring.internal.RingVideoServlet;
+import org.openhab.binding.ring.internal.data.Profile;
+import org.openhab.binding.ring.internal.data.RingDevices;
+import org.openhab.binding.ring.internal.data.RingEvent;
+import org.openhab.binding.ring.internal.errors.AuthenticationException;
+import org.openhab.binding.ring.internal.errors.DuplicateIdException;
+import org.openhab.binding.ring.internal.utils.RingUtils;
+import org.openhab.core.OpenHAB;
+import org.openhab.core.config.core.Configuration;
+import org.openhab.core.library.types.DateTimeType;
+import org.openhab.core.library.types.OnOffType;
+import org.openhab.core.library.types.StringType;
+import org.openhab.core.net.NetworkAddressService;
+import org.openhab.core.thing.Bridge;
+import org.openhab.core.thing.ChannelUID;
+import org.openhab.core.thing.ThingStatus;
+import org.openhab.core.thing.ThingStatusDetail;
+import org.openhab.core.thing.binding.BaseBridgeHandler;
+import org.openhab.core.types.Command;
+import org.openhab.core.types.RefreshType;
+import org.osgi.service.http.HttpService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.google.gson.JsonParseException;
+
+/**
+ * The {@link RingDoorbellHandler} is responsible for handling commands, which are
+ * sent to one of the channels.
+ *
+ * @author Wim Vissers - Initial contribution
+ * @author Peter Mietlowski - oAuth upgrade and additional maintenance
+ * @author Ben Rosenblum - Updated for OH4 / New Maintainer
+ */
+
+@NonNullByDefault
+public class AccountHandler extends BaseBridgeHandler implements RingAccount {
+
+    private @Nullable ScheduledFuture<?> jobTokenRefresh = null;
+    private @Nullable ScheduledFuture<?> eventRefresh = null;
+    private @Nullable Runnable runnableVideo = null;
```

Comment:
please use a lambda instead of a `Runnable`

Reply:
OK fixed further down in the code - I think this stays as is.